astropy/time/tests/test_basic.py

```diff
@@ -11,7 +11,7 @@
 
 from ...tests.helper import catch_warnings
 from ...utils import isiterable
-from .. import Time, ScaleValueError, TIME_SCALES, TimeString, TimezoneInfo
+from .. import Time, ScaleValueError, TIME_SCALES, TIME_TYPES, TimeString, TimezoneInfo
```

Comment:
As noted, I think `TIME_TYPES` should not be exported (since `TIME_DELTA_TYPES` is not either). Also, I think the tests are clearer if they do not use `TIME_TYPES['tt']` but rather `STANDARD_TIME_SCALES`. So,
```
from ..core import STANDARD_TIME_SCALES
```